src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -754,4 +747,23 @@ public LogLevel GetConfiguredLogLevel(string loggerFilter = "")
 
         return LogLevel.Error;
     }
+
+    /// <summary>
+    /// Checks if the specified DML tool is enabled in MCP runtime options.
+    /// </summary>
+    public bool IsMcpDmlToolEnabled(string toolName)
+    {
+        if (Runtime?.Mcp?.Enabled != true || Runtime.Mcp.DmlTools == null)
```

Comment:
If DmlTools is null, aren't all tools `enabled` by default? (to align with the principle that you get MCP enabled by default?